src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int INDEX0 = 0;
```

Comment:
```suggestion
    private static final int DATE = 0;
```
Also make the titles informative. Here's an example